cmd/util/ledger/reporters/mock/get_state_commitment_func.go

```diff
@@ -3,9 +3,10 @@
 package mock
 
 import (
-	flow "github.com/onflow/flow-go/model/flow"
 	mock "github.com/stretchr/testify/mock"
 
+	flow "github.com/onflow/flow-go/model/flow"
+
 	testing "testing"
```

Comment:
The official CI Linter has a setting to enforce goimports -w -local github.com/onflow/flow-go for the codebase.
My assumption was that this was cached on the GitHub servers, so it did not complain to other files but my new observer Go ones.
Since my local run of goimports triggered many import reordering issues I applied them all in one change.

The GitHub run that triggered the need for the fix:
https://github.com/GetElastech/flow-go/runs/6872723567?check_suite_focus=true

The simple change to fix them all. I suspect protobuf changes will trigger goimports linter errors to fix this again.
https://github.com/GetElastech/flow-go/commit/ff5b5c09bd29f8fa520aafbd99959e3e9d8cb2f4
https://github.com/GetElastech/flow-go/commit/2da823ca2ab28082c1db7576ef00cb044ee6b1f1

Reply:
They have a conflicting format with our Lint rules. Should I add an automatic goimports to the generation step, so that they always comply?
I think it is a random cache that makes these fail only in case of a few test runs.

Reply:
Let me check one more thing regarding this tomorrow. I tried to rerun the mocker, and it did not revert the imports back. It seems that something else might be in play.